mu-plugins/global-fonts/index.php

```diff
@@ -108,36 +106,57 @@ function maybe_preload_font( $preload ) {
 /**
  * Return the details about a specific font face.
  */
-function get_font_urls( $font ) {
+function get_font_url( $font ) {
```

Comment:
Could the list here be built dynamically? Maybe a list of fonts + subsets which are combined?

Reply:
I've pushed a new commit, could you take a look? Thanks!